content/en/blog/hackweek-2024.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Our First Hackweek"
+date: 2024-08-13
+linkTitle: "Our First Hackweek"
+description: "Our First Hackweek"
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+
+Last week was a special week. We had our first hackweek! With everyone in the team connecting from their own corners of the world, Belgium, Greece, Italy and Spain, we came together not in person, but through a shared passion for hacking. Unlike typical team-building events, this hackweek was all about individual exploration—each team member diving into their own projects, driven by personal curiosity and the opportunity to experiment.
+
+Over the course of the week, we set aside our usual tasks to focus entirely on these self-driven projects, tackling challenges we rarely have time to address or exploring new avenues. In this post, I’ll share a glimpse of some of the projects we worked on.
+
+During the first day we got together to brainstorm. Many good ideas came from each of us on what to work on. It lifts my spirit to see that our project while very stable, still has lots of avenues to explore and that everyone in our team is keen on making Kairos a better project.
+
+## Kairos Kiosk & Quiz App by Dimitris
+
+Since we have some upcoming events where we will have a booth, we were thinking of a way to break the ice, start conversations with people and keep them engaged.
+
+One idea was to set up a Kairos Kiosk with an app that quizzes people passing by with questions that introduce them to the ideas behind Kairos. Those who answer correctly will receive a prize. This approach makes it feel like a game while also serving as a conversation starter, depending on the quiz questions.
+
+We had a look and couldn't find an open source tool that would allow us to build such a quiz. Since I have been out of web development for quite some time, I thought it would be a refreshing project. I took the fastest road to success, using tools I was already familiar with, like [gin-gonic](https://gin-gonic.com/), [gorm](https://gorm.io/) and [SQLite](https://www.sqlite.org/).
```

Comment:
This is a sudden switch to first person. Should we keep it as if you are still doing the "narration" ?

Reply:
or somehow make make it clear that the narrator changed here

Reply:
hmm good question, yeah I thought it would be clear that the narrator changed, but maybe it's not the best

Reply:
In the end I put on the title "by Dimitris" so I think it's ok